### PR TITLE
Allow missing python versionsuffix for existing easyconfig files changed in PRs

### PR DESCRIPTION
**Pull request by Flamefire**
Otherwise this would break a lot of ECs

### CODE REVIEW

**Comment by Flamefire** [comment]
This is a solution for the failure in https://github.com/easybuilders/easybuild-easyconfigs/pull/9628 and being tested there.

File: test/easyconfigs/easyconfigs.py [hunk 1]
```diff
@@ -653,6 +659,11 @@ def check_https(self, changed_ecs):
 
     def test_changed_files_pull_request(self):
         """Specific checks only done for the (easyconfig) files that were changed in a pull request."""
+        def get_eb_files_from_diff(filter):
```
Comment (boegel) [inline test/easyconfigs/easyconfigs.py]: @Flamefire `filter` is a Python keyword (see syntax highlighting in the diff), so let's use `filter_char` instead?

File: test/easyconfigs/easyconfigs.py [hunk 2]
```diff
@@ -682,16 +693,18 @@ def test_changed_files_pull_request(self):
...
-                cmd = "git diff --name-only --diff-filter=AM %s...HEAD" % target_branch
-                out, ec = run_cmd(cmd, simple=False)
-                changed_ecs_filenames = [os.path.basename(f) for f in out.strip().split('\n') if f.endswith('.eb')]
-                print("\nList of changed easyconfig files in this PR: %s" % '\n'.join(changed_ecs_filenames))
+                changed_ecs_filenames = get_eb_files_from_diff(filter='M')
+                added_ecs_filenames = get_eb_files_from_diff(filter='A')
+                if changed_ecs_filenames:
+                    print("\nList of changed easyconfig files in this PR: %s" % '\n'.join(changed_ecs_filenames))
+                if added_ecs_filenames:
+                    print("\nList of adeed easyconfig files in this PR: %s" % '\n'.join(added_ecs_filenames))
```
Comment (boegel) [inline test/easyconfigs/easyconfigs.py]: `adeed` -> `added`

File: test/easyconfigs/easyconfigs.py (hunk 1 above)
Comment (Flamefire) [reply test/easyconfigs/easyconfigs.py]: I used diff_filter as per the git command

File: test/easyconfigs/easyconfigs.py (hunk 2 above)
Comment (Flamefire) [reply test/easyconfigs/easyconfigs.py]: fixed

**Pull request merged**